sdk/iot/core/inc/az_iot_sas_token.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef AZ_IOT_SAS_TOKEN_H
+#define AZ_IOT_SAS_TOKEN_H
+
+#include <stdlib.h>
+#include <az_result.h>
+#include <az_span.h>
```

Comment:
I think you only need to include <stdint.h> and not the stdlib.h.

In our headers we include azure stuff first (so that if we forgot to include some header, there's a bigger chance to catch that). Then we put a single spacer line and include standard library headers. This way, when you auto-format the header (Ctrl+K+D in VisualStudio), they are sorted alphabetically, but don't get mixed together.

```
#include <az_result.h>
#include <az_span.h>

#include <stdint.h>
```

Reply:
Fixed.